<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphVisualWindow</class>
 <widget class="QDialog" name="GraphVisualWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>User 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="User2"/>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="Most_Inf">
             <property name="text">
              <string>Most Infulence User</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Most_Act">
             <property name="text">
              <string>Most Active User</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="Mutual">
             <property name="text">
              <string>Get Mutual Users</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="People_You">
             <property name="text">
              <string>People You may Know</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>User 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="User1"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Image</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
